src/services/databaseservice.cpp

```diff
@@ -876,6 +876,11 @@ bool DatabaseService::setupTables() {
         version = 41;
     }
 
+    if (version < 42) {
```

Comment:
This will clash with https://github.com/pbek/QOwnNotes/pull/3006/files#diff-dc0007d20d0985e45c46ab027eb6de6be1c5c8fbc38faf31229eaa428c4bed89R879, but is hard to fix before one of the 2 PRs are ready to be merged.

Reply:
Resolving this since we closed the other PR